index.html

```diff
@@ -1,21 +1,114 @@
-<!doctype html>
-<html>
 <head>
-
     <meta charset="utf-8">
-    <title>&lt;juicy-element&gt;</title>
+    <title>&lt;slot-all&gt;</title>
     <link rel="stylesheet" href="http://juicy.github.io/github-markdown-css/github-markdown.css">
 
     <!-- Imports polyfill -->
     <script src="../webcomponentsjs/webcomponents-lite.js"></script>
 
+    <!-- Imports ace editor for nicer demo -->
+    <link rel="import" href="../juicy-ace-editor/juicy-ace-editor.html">
```

Comment:
I’ve got 404 on this import. Should probably be added to `bower.json` as a dev dependency.